leshan-server-cluster/src/main/java/org/eclipse/leshan/server/cluster/LeshanClusterServer.java

```diff
@@ -77,83 +79,93 @@ public static void main(String[] args) {
 
         // Abort if unexpected options
         if (cl.getArgs().length > 0) {
-            System.out.println("Unexpected option or arguments : " + cl.getArgList());
+            System.err.println("Unexpected option or arguments : " + cl.getArgList());
```

Comment:
I prefer using sys.err for error. I don't consider putting bad option as an error this seems to me a normal use case of a command line tool.